src/components/embedded/embedded-stores/EmbeddedMessagesDataProviderStore.tsx

```diff
@@ -331,6 +345,7 @@ export default class EmbeddedMessagesDataProviderStore implements MessagesDataSt
 		}
 
 		if (messages.length !== 0) {
+			this.resumeMessageIdsNext.updateMessageIdsByMessageList(messages);
```

Comment:
when you iterate over messages in `ResumeMessageIdsStore.updateMessageIdsByMessageList` you will iterate from newest to oldest here overwriting `this.messageIds[`${session}:${direction}`] = messageId;`
shouldnt it be reversed here?

Reply:
Good that you noticed it. I haven't known that MessageSSEChannel reorders messages in case the search direction is `next`. Fixed: https://github.com/th2-net/th2-rpt-viewer/pull/344/commits/6766ad93bef27eabec1c1838dd86cbfe24f58515